src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -71,6 +71,11 @@ public void filterBySmallBusinesses() {
     queryOfDatabase = businessesService.getBigBusinessFromDatabase();
     businesses = businessesService.removeBigBusinessesFromResults(queryOfDatabase);
   }
+  
+  public void findBigBusinessInList () {
+     businessesService.checkNumberOfLocationsOfBusiness();
+     filterBySmallBusinesses();
```

Comment:
I think this function may be called twice -- here and on line 88. Is that intentional?

Reply:
YOU ARE A GENIUS, I fixed this. The program should remove the big business the second that it knows it is one.